content/json-rpc.ts

```diff
@@ -101,40 +101,86 @@ class NSUser {
 
 class NSItem {
   /**
-   * Quick-search for items in Zotero.
+   * Search for items in Zotero.
    *
-   * @param terms  Terms as a single string as typed into the search box in Zotero, or an array of tuples as typed into the advanced search box in Zotero
+   * Examples
+   *
+   * - search('') or search([]): return every entries
+   * - search('Zotero'): quick search for 'Zotero'
+   * - search([['title', 'contains', 'Zotero']]): search for 'Zotero' in the Title
+   * - search([['library', 'is', 'My Library']]): search for entries in 'My Library'
+   *   (this function try to resolve the string 'My Library' into is own libraryId number)
+   * - search([['ignore_feeds']]): custom action for ignoring the feeds
+   * - search([['ignore_feeds'], ['quicksearch-titleCreatorYear', 'contains', 'Zotero']]): quick search for 'Zotero' ignoring the Feeds
+   * - search([['creator', 'contains', 'Johnny'], ['title', 'contains', 'Zotero']]): search for entries with Creator 'Johnny' AND Title 'Zotero'
+   * - search([['joinMode', 'any'], ['creator', 'contains', 'Johnny'], ['title', 'contains', 'Zotero']]): search for entries with Creator 'Johnny' OR Title 'Zotero'
+   * - search([['blockStart'], ['creator', 'contains', 'Johnny'], ['title', 'contains', 'Zotero'], ['blockEnd'], ['creator', 'contains', 'Smith']]): search for entries with (Creator 'Johnny' OR Title 'Zotero') AND (Creator 'Smith')
```

Comment:
From my understanding of the code, `blockStart` doesn't work this way. I could be wrong, though. It seems that `blockStart` is specifically used for quick search queries and has some specially handling in code for this use case. I think to acheive `OR` + `AND` the user would want to use `joinMode` `any` and also pass the `required` param: `[['joinMode', 'any'], ['creator', 'contains', 'Johnny'], ['title', 'contains', 'Zotero'], ['creator', 'contains', 'Smith', true]]`